examples/historic_async.py

```diff
@@ -0,0 +1,126 @@
+import os
+import sys
+import asyncio
+import alpaca_trade_api as tradeapi
+from alpaca_trade_api.rest import TimeFrame, URL
+from alpaca_trade_api.rest_async import gather_with_concurrency, AsyncRest
+import pandas as pd
+from enum import Enum
+
+NY = 'America/New_York'
+
+
+class DataType(str, Enum):
+    Bars = "Bars"
+    Trades = "Trades"
+    Quotes = "Quotes"
+
+
+def get_data_method(data_type: DataType):
+    if data_type == DataType.Bars:
+        return rest.get_bars_async
+    elif data_type == DataType.Trades:
+        return rest.get_trades_async
+    elif data_type == DataType.Quotes:
+        return rest.get_quotes_async
+    else:
+        raise Exception(f"Unsupoported data type: {data_type}")
+
+
+async def get_historic_data_base(symbols, data_type: DataType, start, end,
+                                 timeframe: TimeFrame):
```

Comment:
timeframe make no sense for trades and quotes. For the sake of this example a constant timeframe (or even the default!) for bars would be fine.